diff -uprN libconfig-1.7.2/CMakeLists.txt libconfig-1.7.2_bak/CMakeLists.txt
--- libconfig-1.7.2/CMakeLists.txt	2018-01-06 12:58:31.000000000 +0800
+++ libconfig-1.7.2_bak/CMakeLists.txt	2020-07-16 13:54:42.552468110 +0800
@@ -11,6 +11,10 @@ set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMA
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/out)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/out)
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fPIC")
+set(BUILD_SHARED_LIBS "OFF")
+
 set(LIB_TYPE STATIC)
 if(BUILD_SHARED_LIBS)
 	set(LIB_TYPE SHARED)
diff -uprN libconfig-1.7.2/lib/CMakeLists.txt libconfig-1.7.2_bak/lib/CMakeLists.txt
--- libconfig-1.7.2/lib/CMakeLists.txt	2018-01-06 12:58:31.000000000 +0800
+++ libconfig-1.7.2_bak/lib/CMakeLists.txt	2020-07-16 13:56:54.103833451 +0800
@@ -41,14 +41,14 @@ set_target_properties(libconfig++
 target_compile_definitions(libconfig
     PRIVATE
         _CRT_SECURE_NO_DEPRECATE
-        _STDLIB_H
+        # _STDLIB_H
         YY_NO_UNISTD_H
         YY_USE_CONST )
 
 target_compile_definitions(libconfig++ 
     PRIVATE
         _CRT_SECURE_NO_DEPRECATE
-        _STDLIB_H
+        # _STDLIB_H
         YY_NO_UNISTD_H
         YY_USE_CONST )
 
@@ -69,4 +69,4 @@ install(TARGETS libconfig++
     LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
     RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
     PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
-)
\ No newline at end of file
+)
