Only in libconfig-1.7.2.bak: build
diff -upr libconfig-1.7.2/CMakeLists.txt libconfig-1.7.2.bak/CMakeLists.txt
--- libconfig-1.7.2/CMakeLists.txt	2018-01-06 12:58:31.000000000 +0800
+++ libconfig-1.7.2.bak/CMakeLists.txt	2019-06-06 11:54:28.000000000 +0800
@@ -11,6 +11,10 @@ set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${CMA
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/out)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/out)
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fPIC")
+
+
 set(LIB_TYPE STATIC)
 if(BUILD_SHARED_LIBS)
 	set(LIB_TYPE SHARED)
Only in libconfig-1.7.2.bak: _install
diff -upr libconfig-1.7.2/lib/CMakeLists.txt libconfig-1.7.2.bak/lib/CMakeLists.txt
--- libconfig-1.7.2/lib/CMakeLists.txt	2018-01-06 12:58:31.000000000 +0800
+++ libconfig-1.7.2.bak/lib/CMakeLists.txt	2019-06-06 02:19:03.571636536 +0800
@@ -41,14 +41,14 @@ set_target_properties(libconfig++
 target_compile_definitions(libconfig
     PRIVATE
         _CRT_SECURE_NO_DEPRECATE
-        _STDLIB_H
+       # _STDLIB_H
         YY_NO_UNISTD_H
         YY_USE_CONST )
 
 target_compile_definitions(libconfig++ 
     PRIVATE
         _CRT_SECURE_NO_DEPRECATE
-        _STDLIB_H
+       # _STDLIB_H
         YY_NO_UNISTD_H
         YY_USE_CONST )
 
Only in libconfig-1.7.2.bak: p1.orig
